sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -338,10 +340,12 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 
```

Comment:
@mssfang can you also include some of the key concepts mentioned here - https://github.com/Azure/azure-sdk-for-net/pull/9172/files#diff-c183227058ebd27d819132b62a437f6eR81

Reply:
will do

Reply:
We should have these concepts as well. Since .NET is more compatible with Java, I think it might be a good idea to include the explanations that explain the hierarchy of our output. I will update it in this PR as well.